content/en/docs/setup/other_config/configuration/index.md

```diff
@@ -229,7 +229,7 @@ encrypt:
   key: mykey
 ```
 
-For more information about using the Config Server's encryption and decryption features, see the [Spring Cloud Config Server documentation](https://cloud.spring.io/spring-cloud-static/spring-cloud-config/2.2.1.RELEASE/single/spring-cloud-config.html#_encryption_and_decryption). 
+For more information about using the Config Server's encryption and decryption features, see the [Spring Cloud Config Server documentation](https://docs.spring.io/spring-cloud-config/docs/3.1.1/reference/html/#_encryption_and_decryption). 
```

Comment:
Looks like the old link is broken, so thanks for updating it.

How'd you choose the new link?  From what I can see [this](https://github.com/spinnaker/kork/blob/420f0a3064380380388b596e8eaf4c6a9ffd7555/spinnaker-dependencies/spinnaker-dependencies.gradle#L29) specifies `Hoxton.SR4` and [this](https://repo1.maven.org/maven2/org/springframework/cloud/spring-cloud-dependencies/Hoxton.SR4/spring-cloud-dependencies-Hoxton.SR4.pom) specifies version `2.2.3.RELEASE`, but the oldest version with docs [here](https://docs.spring.io/spring-cloud-config/docs/) is [2.2.4.RELEASE](https://docs.spring.io/spring-cloud-config/docs/2.2.4.RELEASE/reference/html/#_encryption_and_decryption).